test/useLocalStore.test.tsx

```diff
@@ -455,3 +455,69 @@ describe("is used to keep observable within component body", () => {
         restore()
     })
 })
+
+describe("enforcing actions", () => {
+    it("'never' should work", () => {
+        mobx.configure({ enforceActions: 'never' })
+        const restore = mockConsole() // to ignore React showing caught errors
```

Comment:
Are there some actual errors printed? Since all three cases, you expect an error to be undefined, it feels like you are trying to hide something nefarious here :)

Reply:
To be honest I just looked for the simplest existing tests, copied them, and modified them slightly. I'm not sure about React showing errors comment. Looking the test I coped from again, I can see now that it doesn't make sense to use the mock console as we're not testing errors that are exposed to the user. Given the way `renderHooks` returns results I'm not sure how to clarify the intention more than just the undefined check. Here is what I'm thinking:

```ts
it("'always' should work", () => {
    configure({ enforceActions: 'always' })
    const { result } = renderHook(() => useAsObservableSource({ hello: 'world' }))
    expect(result.error).not.toBeDefined();
})
```

![Screen Shot 2019-06-13 at 10 15 22](https://user-images.githubusercontent.com/7257660/59441622-dfd79780-8dc6-11e9-861d-48fdd33cdec6.png)

Reply:
The screenshot is with the new `runInAction` code temporarily removed so I could show the error.

Reply:
Yes, I think that's a much better way like that.